app/jobs/scrape_catalogues_job.rb

```diff
@@ -18,43 +12,18 @@ def perform(catalogue_to_add)
       begin
 
         #visit page
-        visit catalogue_url(catalogue_num)
+        visit catalogue_url(catalogue_num, nil)

```

Comment:
Consider changing the method definition to have a default nil value for `page_num`:

```ruby
def catalogue_url(catalogue_num, page_num=nil)
```

Then this line 15 can become:

```ruby
visit catalogue_url(catalogue_num)
```